ksp_plugin_test/interface_compatibility_test.cpp

```diff
@@ -0,0 +1,124 @@
+﻿
+#include <string>
+#include <vector>
+
+#include "base/file.hpp"
+#include "base/pull_serializer.hpp"
+#include "base/push_deserializer.hpp"
+#include "glog/logging.h"
+#include "gmock/gmock.h"
+#include "gtest/gtest.h"
+#include "ksp_plugin/interface.hpp"
+#include "ksp_plugin/plugin.hpp"
+#include "serialization/ksp_plugin.pb.h"
+#include "testing_utilities/serialization.hpp"
+
+namespace principia {
+namespace interface {
+
+using base::OFStream;
+using base::PullSerializer;
+using base::PushDeserializer;
+using ksp_plugin::Plugin;
+using testing_utilities::ReadLinesFromBase64File;
+using testing_utilities::ReadLinesFromHexadecimalFile;
+using ::testing::NotNull;
+
+const char preferred_compressor[] = "gipfeli";
+const char preferred_encoder[] = "base64";
+
+class InterfaceCompatibilityTest : public testing::Test {
```

Comment:
Let us keep that in PluginCompatibilityTest; while this test is interface-only, we may want to inspect saves in other tests, e.g., https://github.com/eggrobin/Principia/blob/saves/ksp_plugin_test/plugin_compatibility_test.cpp.

Reply:
Done.